runtime/oti/cfr.h

```diff
@@ -923,22 +921,14 @@ typedef struct J9CfrClassFile {
 #define CFR_MINOR_VERSION  3
 #define CFR_PUBLIC_PRIVATE_PROTECTED_MASK	(CFR_ACC_PUBLIC | CFR_ACC_PRIVATE | CFR_ACC_PROTECTED)
 
+#define J9_IS_CLASSFILE_VALUETYPE(classfile) ((classfile->majorVersion >= VALUE_TYPES_MAJOR_VERSION) && (PREVIEW_MINOR_VERSION == classfile->minorVersion) && J9_ARE_NO_BITS_SET(classfile->accessFlags, CFR_ACC_IDENTITY))
```

Comment:
Please change `classfile->majorVersion`, `classfile->minorVersion` and `classfile->accessFlags` to
` (classfile)->majorVersion`, `(classfile)->minorVersion` and `(classfile)->accessFlags`